aiomisc/entrypoint.py

```diff
@@ -31,6 +34,10 @@
 asyncio_current_task = asyncio.current_task
 
 
+def is_main_thread() -> bool:
+    return threading.current_thread().ident == MAIN_THREAD_INDENT
```

Comment:
Maybe it will be safer and/or more appropriate to check against [`threading.main_thread()`](https://docs.python.org/3/library/threading.html#threading.main_thread)?

I'm concerned about the case when `aiomisc` is lazy imported in another thread.

But, I'm not sure what exact problem this PR is solving, so I can't be sure that my suggestion is fitting.